functions/data.json

```diff
@@ -0,0 +1,649 @@
+{
```

Comment:
Unless you need the backend for this (or any PR), no need to commit this file or anything in the `/functions` dir since it isn't used until we need a backend